src/server/auth-token.js

```diff
@@ -61,21 +61,22 @@ class AuthToken {
   static authenticate(req, res, next) {
     const bearerToken = req.headers.authorization;
     if (!bearerToken) {
-      return next();
+      next();
+      return;
```

Comment:
if return is not required when next is called, why do you have return here then?